brewtils/test/comparable.py

```diff
@@ -135,7 +136,7 @@ def _assert_equal(obj1, obj2, expected_type=None, deep_fields=None):
                 deep_fields[key](nested1, nested2)
 
 
-def _assert_wrapper(obj1, obj2, do_raise=False, **kwargs):
+def _assert_wrapper(obj1, obj2, expected_type=None, do_raise=False, **kwargs):
```

Comment:
This is the only actual code change. Sphinx was yelling at me about `expected_type` not being an actual parameter.  It looked like all of calls to this explicitly provide it, so I just made it a proper keyword arg.  The tests still pass and I can't think of any reason why this should be a problem, but feel free to double check.